unit-files/check_health.service

```diff
@@ -0,0 +1,16 @@
+# check_health.service
+[Unit]
+Description=Check Health Service
+Requires=docker.service
+Requires=bitcore.service
+Requires=bitcoin_core.service
+Requires=create_docker_network.service
+
+[Service]
+Type=oneshot
+
+ExecStartPre=-/usr/bin/rm -f /tmp/health-check*
```

Comment:
`/tmp/health-check` i don't see this path defined anywhere, should it be `TEMP_COUNTER_FILE_BASE_NAME="bitcore-health-check"`?

Reply:
Good catch